CHANGELOG.md

```diff
@@ -65,7 +65,7 @@
 
 
 ### 🛡 Security
-
+- update css loader to 5.0.0 ([1530](https://github.com/opensearch-project/oui/pull/1530))
```

Comment:
These are changes for 1.17 - this should be go in the unreleased section, right?